src/core/message-reply.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseMessageSchema } from './types';
+
+type Status = {
+  code: number
+  message: string
+};
+
+type MessageResultOpts = {
```

Comment:
For consistency
```suggestion
type MessageResultOptions = {
```

Reply:
https://github.com/TBD54566975/dwn-sdk-js/commit/79558b120f081869b43b92826b673d788cec72df